Banyan/src/sessions.jl

```diff
@@ -329,6 +329,10 @@ function get_session_status(session_id::String=get_session_id(); kwargs...)
     configure(; kwargs...)
     filters = Dict("session_id" => session_id)
     response = send_request_get_response(:describe_sessions, Dict{String,Any}("filters"=>filters))
+    if !haskey(response["sessions"], session_id)
```

Comment:
What is this case for? Not sure I understand

Reply:
It was a weird edge-case I encountered where a session waiting for creating to finish suddenly had an error that the result of trying to get its updated session status was that the session ID was not in the result of `describe-sessions`. This handles that edge-case with a warning.

Reply:
Ah, and this is because of a bug in describe_sessions, which will be fixed soon